nntrainer/tensor/weight.cpp

```diff
@@ -24,9 +24,8 @@ Weight::Weight(const TensorDim &dim, const WeightInitializer init, bool train,
     throw std::invalid_argument("Weight initializer unknown");
 }
 
-void Weight::initialize(const Tensor &weights_preallocated,
-                        const Tensor &grad_preallocated) {
-  Var_Grad::initialize(weights_preallocated, grad_preallocated);
+void Weight::initializeWeight(const Tensor &weights_preallocated, bool gtrain) {
+  Var_Grad::initializeWeight(weights_preallocated, gtrain);
```

Comment:
do we need gtrain?

Reply:
same as above - to maintain same interface as `initializeGradient`